src/modalities/__main__.py

```diff
@@ -34,12 +36,59 @@
 from modalities.trainer import Trainer
 from modalities.util import compute_number_of_trainable_parameters
 
+_ROOT_DIR = Path(__file__).parents[2]
```

Comment:
Good point. The problem is solved in https://github.com/Modalities/modalities/pull/155/commits/563cf27bd1f5bca5fe83caf25fd8f52b68f44421 by using a script instead of a CLI.